core/drivers/stm32_bsec.c

```diff
@@ -616,8 +616,41 @@ static void bsec_dt_otp_nsec_access(void *fdt, int bsec_node)
 			}
 		}
 
-		if (!fdt_getprop(fdt, bsec_subnode, "st,non-secure-otp", NULL))
+		/* Handle different kinds of non-secure accesses */
+		if (fdt_getprop(fdt, bsec_subnode,
+				"st,non-secure-otp-provisioning", NULL)) {
+			bool locked = false;
+			bool lcked_next = false;
```

Comment:
s/lcked_next/locked_next/g

Reply:
Change to locked_2. I did this because of character limitation